style.css

```diff
@@ -25,7 +25,6 @@ html, body {
 /* Inventory */
 
 .inventory {
-  pointer-events: none;

```

Comment:
this is actually needed for tiny screen sizes, because it's overlapping the refresh button sometimes.

we could keep this line and add the following bit to make both work.

```
.inventoryEntry {
  pointer-events: auto;
```